vcpkg/readme/vcpkg-README.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Microsoft/vcpkg README
+description: README for vcpkg registry
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 10/17/2023
+ms.prod: vcpkg
+ROBOTS: NOINDEX
+---
+
+# vcpkg: Overview
+
+vcpkg helps you manage C and C++ libraries on Windows, Linux and MacOS.
+This tool and ecosystem are constantly evolving, and we always appreciate contributions!
+
+If you've never used vcpkg before, or if you're trying to figure out how to use vcpkg,
+check out our [guide to start using vcpkg](../get_started/get-started.md).
+
+For a short description of all available commands, run `vcpkg help`. Run `vcpkg help [command name]` for help on a specific command.
+
+* GitHub: ports at [Microsoft/vcpkg](<https://github.com/microsoft/vcpkg>), program at [Microsoft/vcpkg-tool](<https://github.com/microsoft/vcpkg-tool>)
+* Slack: [C++ Alliance's Slack server](<https://cppalliance.org/slack/>), in the #vcpkg channel
+* Discord: [\#include \<C++\>'s Discord server](<https://www.includecpp.org>), in the #🌏vcpkg channel
+* Docs: [Documentation](/vcpkg)
+
+# Getting Started
+
+First, follow the [quick start guide](../get_started/get-started.md).
+
+If a library you need is not present in the vcpkg catalog, you can [open an issue on the GitHub repo][contributing:submit-issue] where the vcpkg team and community can see it, and potentially add the port to vcpkg.
+
+After you've gotten vcpkg installed and working, you may wish to add [tab completion](#tab-completionauto-completion) to your shell.
+
+## Quick Start: Windows
```

Comment:
I think we should just link to our getting started documentation which is more up to date and includes manifest mode instructions. This suggestion applies to all the OS pivots here.